dataframe/field_config.go

```diff
@@ -0,0 +1,70 @@
+package dataframe
+
+// FieldConfig represents the display properties for a field
+type FieldConfig struct {
+	Title      string `json:"title,omitempty"`
+	Filterable *bool  `json:"filterable,omitempty"`
+
+	// Numeric Options
+	unit     string `json:"title,omitempty"`
```

Comment:
Fields (properties) that are in a struct that are also not-exported (not-capitalized) will be ignored in JSON marshaling and unmarshaling.